StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheetConfiguration.swift

```diff
@@ -81,6 +81,11 @@ extension CustomerSheet {
         /// Note: Card brand filtering is not currently supported by Link.
         @_spi(CardBrandFilteringBeta) public var cardBrandAcceptance: PaymentSheet.CardBrandAcceptance = .all
 
+        /// This is an experimental feature that may be removed at any time.
+        /// If true, when editing, cards and us bank accounts will have the edit icon, users cannot remove them from the list screen.
+        /// If false (default), only cbc eligible cards can be edited and users can remove payment methods from the list screen.
+        @_spi(DefaultSPM) public var defaultSPMNavigation = false
```

Comment:
Can we find a better name for this? I don't think `DefaultSPM` and `defaultSPMNavigation` really describe what this flag is doing. It's also odd that it is "default" but by default is turned off, and turning it on results in non-default behavior.

Reply:
Let's also replace CBC with card brand choice cards, merchants may not know what "cbc" is.

Reply:
I'm not really sure of a better name— it seems like we'd need two separate flags for this project: one specifically to trigger the updated navigation flow and one to opt in to allowing default SPMs. The reasoning I had for why there would be two flags is for the case that would lead us to the remove-only screen, in which case the navigation flag would be on and the set as default flag would be off. My thought was that the navigation flag would eventually be removed altogether as the new navigation flow would completely replace the old one. Maybe there's a better way though

Reply:
Would newUpdatePaymentMethodNavigation be better?

Reply:
I've refactored it to alternateUpdatePaymentMethodNavigation, but I'm open to any other change suggestions